frontend/src/components/place/AddPlace.vue

```diff
@@ -394,6 +378,10 @@ export default {
         'newPlace.category': function (categoryObject) {
             if (categoryObject) {
```

Comment:
Better to write logic vie versa here
if (_.isEmpty(categoryObject)) { return;  }
... // logic